packages/timeline-state-resolver/src/integrations/vizMSE/$schemas/actions.json

```diff
@@ -7,6 +7,13 @@
 			"description": "Purges all elements from the viz-rundown",
 			"destructive": true,
 			"timeout": 10000
+		},
+		{
+			"id": "clearAllEngines",
+			"name": "Clear Engines",
+			"description": "Clears all Engines in the Profile by sending the configured list of clear-commands",
+			"destructive": true,
```

Comment:
I'm not sure what `destructive` means, so I yoloed on `true`

Reply:
It is a UX thing. True is good.

Reply:
yes, `true` is correct here. `destructive` means that the action can affect playout (ie "use with caution").